pyproject.toml

```diff
@@ -7,7 +7,8 @@ license = { text = "MIT" }
 requires-python = ">=3.8"
 dependencies = [
     "flask>=1.1.2",
-    "requests>=2.25.1"
+    "requests>=2.25.1",
+    "flask_cors"
 ]
 

```

Comment:
Can you also add a step that verifies the installation? This can be done right after `pdm install -d`:

`- run:                           `
`    name: List installed packages`
`     command: pdm list             `